kits/arm/example_AR_kit.cpp

```diff
@@ -0,0 +1,200 @@
+/*
+ * AR Kit Example
+ * Currently assumes 6 degrees of freedom in you arm!
+ */
+
+#include "group_command.hpp"
+#include "group_feedback.hpp"
+#include "robot_model.hpp"
+#include "arm/arm.hpp"
+#include "util/mobile_io.hpp"
+#include <chrono>
+#include <iostream>
+
+using namespace hebi;
+using namespace experimental; // For all things mobileIO 
+
+
+
+Eigen::Matrix3d makeRotationMatrix (hebi::Quaternionf phone_orientation) {
+  Eigen::Quaterniond q;
+  q.w() = phone_orientation.getW();
+  q.x() = phone_orientation.getX();
+  q.y() = phone_orientation.getY();
+  q.z() = phone_orientation.getZ();
+  return q.toRotationMatrix();
+}
```

Comment:
(future C++ API) Eventually I'd love for this to be an optional return value from the hebi::Quaternionf object...